server/src/main/java/org/elasticsearch/script/CompositeFieldScript.java

```diff
@@ -37,7 +38,7 @@ public interface LeafFactory {
     private final Map<String, List<Object>> fieldValues = new HashMap<>();
 
     public CompositeFieldScript(String fieldName, Map<String, Object> params, SearchLookup searchLookup, LeafReaderContext ctx) {
-        super(fieldName, params, searchLookup, ctx);
+        super(fieldName, params, searchLookup, ctx, ErrorBehaviour.FAIL);
```

Comment:
the flag should be supported in the definition of the composite runtime field too, then carried around all the way here I think.

Reply:
added support for composite now